[2020-09-10-00:03:04]from sport : <class 'requests.exceptions.ReadTimeout'>HTTPSConnectionPool(host='sport.le360.ma', port=443): Read timed out. (read timeout=5)<traceback object at 0x110A1F48>
[2020-09-10-00:34:29]from sport : <class 'requests.exceptions.ReadTimeout'>HTTPSConnectionPool(host='sport.le360.ma', port=443): Read timed out. (read timeout=5)<traceback object at 0x13741D08>
[2020-09-10-00:49:59]from sport : <class 'requests.exceptions.ReadTimeout'>HTTPSConnectionPool(host='sport.le360.ma', port=443): Read timed out. (read timeout=5)<traceback object at 0x113C5108>
[2020-09-10-13:01:53]from leseco : <class 'requests.exceptions.ReadTimeout'>HTTPSConnectionPool(host='leseco.ma', port=443): Read timed out. (read timeout=5)<traceback object at 0x0171EE68>
[2020-09-10-13:03:11]from sport : <class 'requests.exceptions.ReadTimeout'>HTTPSConnectionPool(host='sport.le360.ma', port=443): Read timed out. (read timeout=5)<traceback object at 0x00E88568>
[2020-09-10-13:03:12]from afrique : <class 'requests.exceptions.ReadTimeout'>HTTPConnectionPool(host='afrique.le360.ma', port=80): Read timed out. (read timeout=5)<traceback object at 0x121E0588>
[2020-09-10-15:02:27]from sport : <class 'requests.exceptions.ReadTimeout'>HTTPSConnectionPool(host='sport.le360.ma', port=443): Read timed out. (read timeout=5)<traceback object at 0x1179EFA8>
[2020-09-10-15:02:29]from le360 : <class 'requests.exceptions.ReadTimeout'>HTTPSConnectionPool(host='fr.le360.ma', port=443): Read timed out. (read timeout=5)<traceback object at 0x11741F08>
[2020-09-10-15:14:51]from sport : <class 'requests.exceptions.ReadTimeout'>HTTPSConnectionPool(host='sport.le360.ma', port=443): Read timed out. (read timeout=5)<traceback object at 0x00B15888>
[2020-09-10-16:34:51]from le360 : <class 'django.db.utils.IntegrityError'>ERREUR:  une valeur NULL viole la contrainte NOT NULL de la colonne « article »
DETAIL:  La ligne en échec contient (79, t, f, f, f, t, f, f, f, f, f, f, f, f, f, f, f, f, f, f, f, f, f, f, f, f, f, f, f, f, f, f, f, f, f, f, f, null).
<traceback object at 0x117F3DE8>
[2020-09-10-16:40:59]from le360 : <class 'requests.exceptions.ReadTimeout'>HTTPSConnectionPool(host='fr.le360.ma', port=443): Read timed out. (read timeout=5)<traceback object at 0x110465E8>
[2020-09-10-17:09:33]from le360 : <class 'django.db.utils.IntegrityError'>ERREUR:  une valeur NULL viole la contrainte NOT NULL de la colonne « article »
DETAIL:  La ligne en échec contient (80, t, f, f, f, t, f, f, f, f, f, f, f, f, f, f, f, f, f, f, f, f, f, f, f, f, f, f, f, f, f, f, f, f, f, f, f, null).
<traceback object at 0x0136FB08>
[2020-09-10-17:27:45]from hespress : <class 'django.db.utils.IntegrityError'>ERREUR:  une valeur NULL viole la contrainte NOT NULL de la colonne « article »
DETAIL:  La ligne en échec contient (81, f, f, f, f, f, f, f, f, f, f, f, f, f, f, f, f, f, f, f, f, f, f, f, f, f, t, f, f, f, f, f, f, f, f, f, f, null).
<traceback object at 0x1142B7A8>
[2020-09-10-18:03:14]from sport : <class 'requests.exceptions.ReadTimeout'>HTTPSConnectionPool(host='sport.le360.ma', port=443): Read timed out. (read timeout=5)<traceback object at 0x0100BDE8>
[2020-09-10-18:27:41]from sport : <class 'requests.exceptions.ReadTimeout'>HTTPSConnectionPool(host='sport.le360.ma', port=443): Read timed out. (read timeout=5)<traceback object at 0x0102EE88>
[2020-09-10-18:28:05]from hespress : <class 'requests.exceptions.ReadTimeout'>HTTPSConnectionPool(host='fr.hespress.com', port=443): Read timed out. (read timeout=5)<traceback object at 0x00FBEE28>
[2020-09-10-18:30:05]from le360 : <class 'django.db.utils.IntegrityError'>ERREUR:  une valeur NULL viole la contrainte NOT NULL de la colonne « article »
DETAIL:  La ligne en échec contient (82, t, f, f, f, t, f, f, f, f, f, f, f, f, f, f, f, f, f, f, f, f, f, f, f, f, f, f, f, f, f, f, f, f, f, f, f, null).
<traceback object at 0x11ADF4E8>
[2020-09-10-19:13:33]from sport : <class 'requests.exceptions.ReadTimeout'>HTTPSConnectionPool(host='sport.le360.ma', port=443): Read timed out. (read timeout=5)<traceback object at 0x113FE928>
[2020-09-10-19:20:52]from hespress : <class 'django.db.utils.IntegrityError'>ERREUR:  une valeur NULL viole la contrainte NOT NULL de la colonne « article »
DETAIL:  La ligne en échec contient (83, f, f, f, f, f, f, f, f, f, f, f, f, f, f, f, f, f, f, f, f, f, f, f, f, f, t, f, f, f, f, f, f, f, f, f, f, null).
<traceback object at 0x010F63A8>
[2020-09-10-19:24:56]from hespress : <class 'django.db.utils.ProgrammingError'>ERREUR:  la colonne « article » de la relation « home_category » n'existe pas
LINE 1: ... "simp_comp", "title", "image", "source", "date", "article")...
                                                             ^
<traceback object at 0x1380F728>
[2020-09-10-19:25:57]from hespress : <class 'django.db.utils.ProgrammingError'>ERREUR:  la colonne « article » de la relation « home_category » n'existe pas
LINE 1: ... "simp_comp", "title", "image", "source", "date", "article")...
                                                             ^
<traceback object at 0x115E4148>
[2020-09-10-19:26:58]from hespress : <class 'django.db.utils.ProgrammingError'>ERREUR:  la colonne « article » de la relation « home_category » n'existe pas
LINE 1: ... "simp_comp", "title", "image", "source", "date", "article")...
                                                             ^
<traceback object at 0x01129488>
[2020-09-10-19:27:59]from hespress : <class 'django.db.utils.ProgrammingError'>ERREUR:  la colonne « article » de la relation « home_category » n'existe pas
LINE 1: ... "simp_comp", "title", "image", "source", "date", "article")...
                                                             ^
<traceback object at 0x0E719B28>
[2020-09-10-20:51:00]from le360 : <class 'requests.exceptions.ReadTimeout'>HTTPSConnectionPool(host='fr.le360.ma', port=443): Read timed out. (read timeout=5)<traceback object at 0x0150A9C8>
[2020-09-10-20:56:09]from le360 : <class 'requests.exceptions.ReadTimeout'>HTTPSConnectionPool(host='fr.le360.ma', port=443): Read timed out. (read timeout=5)<traceback object at 0x018F5B08>
[2020-09-10-21:49:12]from sport : <class 'requests.exceptions.ReadTimeout'>HTTPSConnectionPool(host='sport.le360.ma', port=443): Read timed out. (read timeout=5)<traceback object at 0x016D2D68>
[2020-09-10-22:04:39]from sport : <class 'requests.exceptions.ReadTimeout'>HTTPSConnectionPool(host='sport.le360.ma', port=443): Read timed out. (read timeout=5)<traceback object at 0x118015A8>
[2020-09-10-22:19:58]from sport : <class 'requests.exceptions.ReadTimeout'>HTTPSConnectionPool(host='sport.le360.ma', port=443): Read timed out. (read timeout=5)<traceback object at 0x016338A8>
[2020-09-10-22:35:12]from sport : <class 'requests.exceptions.ReadTimeout'>HTTPSConnectionPool(host='sport.le360.ma', port=443): Read timed out. (read timeout=5)<traceback object at 0x11D7AFA8>
[2020-09-10-23:10:06]from le360 : <class 'NameError'>name 'unicodedata' is not defined<traceback object at 0x13874E48>
[2020-09-10-23:11:09]from le360 : <class 'NameError'>name 'unicodedata' is not defined<traceback object at 0x00F77C88>
[2020-09-10-23:12:10]from le360 : <class 'NameError'>name 'unicodedata' is not defined<traceback object at 0x0274D6E8>
[2020-09-10-23:13:12]from le360 : <class 'NameError'>name 'unicodedata' is not defined<traceback object at 0x13694708>
[2020-09-10-23:14:14]from le360 : <class 'NameError'>name 'unicodedata' is not defined<traceback object at 0x136DF528>
[2020-09-10-23:15:16]from le360 : <class 'NameError'>name 'unicodedata' is not defined<traceback object at 0x11662E48>
[2020-09-10-23:16:18]from le360 : <class 'NameError'>name 'unicodedata' is not defined<traceback object at 0x0274FAE8>
[2020-09-10-23:17:09]from hespress : <class 'NameError'>name 'unicodedata' is not defined<traceback object at 0x0179B4E8>
[2020-09-10-23:17:15]from sport : <class 'NameError'>name 'unicodedata' is not defined<traceback object at 0x112E4BA8>
[2020-09-10-23:17:18]from leseco : <class 'NameError'>name 'unicodedata' is not defined<traceback object at 0x0333D4C8>
[2020-09-10-23:17:21]from le360 : <class 'NameError'>name 'unicodedata' is not defined<traceback object at 0x1151F9E8>
[2020-09-10-23:18:15]from hespress : <class 'NameError'>name 'unicodedata' is not defined<traceback object at 0x1417D4C8>
[2020-09-10-23:18:15]from welovebuzz : <class 'NameError'>name 'unicodedata' is not defined<traceback object at 0x13F6F788>
[2020-09-10-23:18:15]from le360 : <class 'NameError'>name 'unicodedata' is not defined<traceback object at 0x1359DA28>
[2020-09-10-23:18:15]from leseco : <class 'NameError'>name 'unicodedata' is not defined<traceback object at 0x1375FCE8>
[2020-09-10-23:18:16]from afrique : <class 'NameError'>name 'unicodedata' is not defined<traceback object at 0x13BD3A68>
[2020-09-10-23:18:19]from sport : <class 'NameError'>name 'unicodedata' is not defined<traceback object at 0x14124B28>
[2020-09-10-23:19:16]from welovebuzz : <class 'NameError'>name 'unicodedata' is not defined<traceback object at 0x01B62708>
[2020-09-10-23:19:17]from le360 : <class 'NameError'>name 'unicodedata' is not defined<traceback object at 0x02CF38C8>
[2020-09-10-23:19:17]from hespress : <class 'NameError'>name 'unicodedata' is not defined<traceback object at 0x11BDEEE8>
[2020-09-10-23:19:18]from leseco : <class 'NameError'>name 'unicodedata' is not defined<traceback object at 0x112EDEE8>
[2020-09-10-23:19:19]from afrique : <class 'NameError'>name 'unicodedata' is not defined<traceback object at 0x1175FAA8>
[2020-09-10-23:19:20]from sport : <class 'NameError'>name 'unicodedata' is not defined<traceback object at 0x0EFAD568>
[2020-09-10-23:19:56]from afrique : <class 'NameError'>name 'unicodedata' is not defined<traceback object at 0x13D139E8>
[2020-09-10-23:19:56]from welovebuzz : <class 'NameError'>name 'unicodedata' is not defined<traceback object at 0x13E0F748>
[2020-09-10-23:19:56]from hespress : <class 'NameError'>name 'unicodedata' is not defined<traceback object at 0x1398D4E8>
[2020-09-10-23:19:57]from le360 : <class 'NameError'>name 'unicodedata' is not defined<traceback object at 0x1379D9C8>
[2020-09-10-23:19:59]from leseco : <class 'NameError'>name 'unicodedata' is not defined<traceback object at 0x1382ECA8>
[2020-09-10-23:20:00]from sport : <class 'NameError'>name 'unicodedata' is not defined<traceback object at 0x135F4D68>
[2020-09-11-10:22:51]from sport : <class 'requests.exceptions.ReadTimeout'>HTTPSConnectionPool(host='sport.le360.ma', port=443): Read timed out. (read timeout=5)<traceback object at 0x11B71E48>
[2020-09-11-10:38:03]from sport : <class 'requests.exceptions.ReadTimeout'>HTTPSConnectionPool(host='sport.le360.ma', port=443): Read timed out. (read timeout=5)<traceback object at 0x034CD2A8>
[2020-09-11-12:00:04]from le360 : <class 'requests.exceptions.ReadTimeout'>HTTPSConnectionPool(host='fr.le360.ma', port=443): Read timed out. (read timeout=5)<traceback object at 0x117DC8C8>
[2020-09-11-12:10:15]from le360 : <class 'requests.exceptions.ReadTimeout'>HTTPSConnectionPool(host='fr.le360.ma', port=443): Read timed out. (read timeout=5)<traceback object at 0x11835528>
[2020-09-11-13:11:30]from sport : <class 'requests.exceptions.ReadTimeout'>HTTPSConnectionPool(host='sport.le360.ma', port=443): Read timed out. (read timeout=5)<traceback object at 0x116C8A48>
[2020-09-11-14:10:31]from sport : <class 'requests.exceptions.ReadTimeout'>HTTPSConnectionPool(host='sport.le360.ma', port=443): Read timed out. (read timeout=5)<traceback object at 0x13DAA048>
[2020-09-11-14:54:57]from le360 : <class 'requests.exceptions.ReadTimeout'>HTTPSConnectionPool(host='fr.le360.ma', port=443): Read timed out. (read timeout=5)<traceback object at 0x13DC4DE8>
[2020-09-11-15:00:04]from le360 : <class 'requests.exceptions.ReadTimeout'>HTTPSConnectionPool(host='fr.le360.ma', port=443): Read timed out. (read timeout=5)<traceback object at 0x03339828>
[2020-09-11-15:14:02]from sport : <class 'requests.exceptions.ReadTimeout'>HTTPSConnectionPool(host='sport.le360.ma', port=443): Read timed out. (read timeout=5)<traceback object at 0x034F6588>
[2020-09-11-16:15:41]from leseco : <class 'django.db.utils.DataError'>ERREUR:  valeur trop longue pour le type character varying(10000)
<traceback object at 0x00FF1628>
[2020-09-11-20:19:59]from sport : <class 'requests.exceptions.ReadTimeout'>HTTPSConnectionPool(host='sport.le360.ma', port=443): Read timed out. (read timeout=5)<traceback object at 0x00A01588>
[2020-09-11-20:35:11]from sport : <class 'requests.exceptions.ReadTimeout'>HTTPSConnectionPool(host='sport.le360.ma', port=443): Read timed out. (read timeout=5)<traceback object at 0x00978A68>
[2020-09-11-20:50:28]from sport : <class 'requests.exceptions.ReadTimeout'>HTTPSConnectionPool(host='sport.le360.ma', port=443): Read timed out. (read timeout=5)<traceback object at 0x00CC60A8>
[2020-09-11-21:20:52]from sport : <class 'requests.exceptions.ReadTimeout'>HTTPSConnectionPool(host='sport.le360.ma', port=443): Read timed out. (read timeout=5)<traceback object at 0x00978248>
[2020-09-11-22:13:30]from sport : <class 'requests.exceptions.ReadTimeout'>HTTPSConnectionPool(host='sport.le360.ma', port=443): Read timed out. (read timeout=5)<traceback object at 0x10FB98C8>
[2020-09-12-07:18:21]from sport : <class 'requests.exceptions.ChunkedEncodingError'>("Connection broken: ConnectionResetError(10054, 'Une connexion existante a dû être fermée par lhôte distant', None, 10054, None)", ConnectionResetError(10054, 'Une connexion existante a dû être fermée par lhôte distant', None, 10054, None))<traceback object at 0x01096508>
[2020-09-12-11:11:50]from afrique : <class 'django.db.utils.DataError'>ERREUR:  valeur trop longue pour le type character varying(10000)
<traceback object at 0x01789CE8>
[2020-09-12-13:00:19]from afrique : <class 'AttributeError'>'NoneType' object has no attribute 'find'<traceback object at 0x112C7FC8>
[2020-09-12-13:01:20]from afrique : <class 'AttributeError'>'NoneType' object has no attribute 'find'<traceback object at 0x00E0E568>
[2020-09-12-13:02:21]from afrique : <class 'AttributeError'>'NoneType' object has no attribute 'find'<traceback object at 0x01718B28>
[2020-09-12-13:03:22]from afrique : <class 'AttributeError'>'NoneType' object has no attribute 'find'<traceback object at 0x11828E48>
[2020-09-12-13:04:23]from afrique : <class 'AttributeError'>'NoneType' object has no attribute 'find'<traceback object at 0x1133AE68>
[2020-09-12-13:05:24]from afrique : <class 'AttributeError'>'NoneType' object has no attribute 'find'<traceback object at 0x01625928>
[2020-09-12-13:06:25]from afrique : <class 'AttributeError'>'NoneType' object has no attribute 'find'<traceback object at 0x0154F268>
[2020-09-12-13:07:26]from afrique : <class 'AttributeError'>'NoneType' object has no attribute 'find'<traceback object at 0x118F1688>
[2020-09-12-13:08:27]from afrique : <class 'AttributeError'>'NoneType' object has no attribute 'find'<traceback object at 0x1180FBA8>
[2020-09-12-13:09:28]from afrique : <class 'AttributeError'>'NoneType' object has no attribute 'find'<traceback object at 0x0154F988>
[2020-09-12-13:10:29]from afrique : <class 'AttributeError'>'NoneType' object has no attribute 'find'<traceback object at 0x017CCCC8>
[2020-09-12-15:35:12]from le360 : <class 'TypeError'>'NoneType' object is not subscriptable<traceback object at 0x1161AC68>
[2020-09-12-15:36:13]from le360 : <class 'TypeError'>'NoneType' object is not subscriptable<traceback object at 0x027FE168>
[2020-09-12-15:37:15]from le360 : <class 'TypeError'>'NoneType' object is not subscriptable<traceback object at 0x00811D08>
[2020-09-12-15:38:16]from le360 : <class 'TypeError'>'NoneType' object is not subscriptable<traceback object at 0x0E3AD0A8>
[2020-09-12-15:39:16]from le360 : <class 'TypeError'>'NoneType' object is not subscriptable<traceback object at 0x0275FF08>
[2020-09-12-15:40:17]from le360 : <class 'TypeError'>'NoneType' object is not subscriptable<traceback object at 0x11025248>
[2020-09-12-15:41:18]from le360 : <class 'TypeError'>'NoneType' object is not subscriptable<traceback object at 0x10A8D4C8>
[2020-09-12-15:42:19]from le360 : <class 'TypeError'>'NoneType' object is not subscriptable<traceback object at 0x1106DB08>
[2020-09-12-15:43:19]from le360 : <class 'TypeError'>'NoneType' object is not subscriptable<traceback object at 0x02770168>
[2020-09-12-15:44:20]from le360 : <class 'TypeError'>'NoneType' object is not subscriptable<traceback object at 0x10A22A28>
[2020-09-12-15:45:21]from le360 : <class 'TypeError'>'NoneType' object is not subscriptable<traceback object at 0x10A37388>
[2020-09-12-15:46:22]from le360 : <class 'TypeError'>'NoneType' object is not subscriptable<traceback object at 0x115F3DA8>
[2020-09-12-15:47:22]from le360 : <class 'TypeError'>'NoneType' object is not subscriptable<traceback object at 0x0E2C0188>
[2020-09-12-15:48:23]from le360 : <class 'TypeError'>'NoneType' object is not subscriptable<traceback object at 0x02760A48>
